package.json

```diff
@@ -9,7 +9,8 @@
   "dependencies": {
     "angular": "^1.5.7",
     "angular-resource": "^1.5.7",
-    "angular-ui-router": "^0.3.1"
+    "angular-ui-router": "^0.3.1",
+    "normalize-sass": "^1.0.0"
```

Comment:
In #12 I pulled in `normalize.css`. I see no reason to use a sass port when the original can compile into the sass just as easily.